developer-workflow/c-api.rst

```diff
@@ -375,14 +375,14 @@ Adding a new definition to the Limited API
 
   .. code-block:: shell
 
-     ./python ./Tools/scripts/stable_abi.py --generate-all ./Misc/stable_abi.toml
+     ./python ./Tools/build/stable_abi.py --generate-all ./Misc/stable_abi.toml
 
 - Build Python and check the using ``make check-limited-abi``.
   On platforms without ``make``, run this command directly:
 
   .. code-block:: shell
 
-    ./python ./Tools/scripts/stable_abi.py --all ./Misc/stable_abi.toml
+    ./python ./Tools/build/stable_abi.py --all ./Misc/stable_abi.toml
```

Comment:
This one still fails on my machine with a fresh build of 3.12 FYI:

```
> python ./Tools/build/stable_abi.py --all ./Misc/stable_abi.toml
Running Debug|x64 interpreter...
Traceback (most recent call last):
  File "C:\Users\alexw\coding\cpython\Tools\build\stable_abi.py", line 757, in <module>
    main()
  File "C:\Users\alexw\coding\cpython\Tools\build\stable_abi.py", line 723, in main
    results['unixy_check'] = do_unixy_check(manifest, args)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\alexw\coding\cpython\Tools\build\stable_abi.py", line 394, in do_unixy_check
    present_macros = gcc_get_limited_api_macros(['Include/Python.h'])
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\alexw\coding\cpython\Tools\build\stable_abi.py", line 526, in gcc_get_limited_api_macros
    sysconfig.get_config_var("CC").split()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'split'
```

But at least the instructions now point to scripts that actually exist 😆